modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -1028,29 +1027,37 @@ class InstructionsTest : IrohaTest<Iroha2Client>() {
     @Story("Account transfers domain ownership")
     @SdkTestId("transfer_domain_ownership")
     fun `transfer domain ownership`(): Unit = runBlocking {
-        val genesisAccountId = AccountId(GENESIS.asDomainId(), GENESIS.asName())
-        var domain = QueryBuilder.findDomainById(DEFAULT_DOMAIN_ID)
+        val domainId = "Kingdom".asDomainId()
+
+        client.sendTransaction {
+            account(super.account)
+            registerDomain(domainId)
+            buildSigned(super.keyPair)
+        }.also { d ->
+            withTimeout(txTimeout) { d.await() }
+        }
+        val aliceDomain = QueryBuilder.findDomainById(domainId)
             .account(super.account)
             .buildSigned(super.keyPair)
             .let { query ->
                 client.sendQuery(query)
             }
-        assertEquals(genesisAccountId, domain.ownedBy)
+        assertEquals(ALICE_ACCOUNT_ID, aliceDomain.ownedBy)
 
         client.tx {
             transferDomainOwnership(
-                genesisAccountId,
-                IdBox.DomainId(DEFAULT_DOMAIN_ID),
+                ALICE_ACCOUNT_ID,
+                IdBox.DomainId(domainId),
                 BOB_ACCOUNT_ID,
             )
         }
-        domain = QueryBuilder.findDomainById(DEFAULT_DOMAIN_ID)
+        val bobDomain = QueryBuilder.findDomainById(domainId)
```

Comment:
same here

Reply:
can you suggest more appropriate names for the 1st domain and the 2nd domain? Kotlin will not allow me to use the same name

Reply:
@mversic in order to reuse the name just change `val` to `var`, the second assignment will happen without declaration
```
var smth = ..
...
smth = ..
```

Reply:
see the suggestion in the comment below